driver-core/src/main/java/com/datastax/driver/core/CBUtil.java

```diff
@@ -117,10 +118,47 @@ public static void writeBytes(byte[] bytes, ByteBuf cb) {
         cb.writeBytes(bytes);
     }
 
+    public static void writeBytes(ByteBuffer bytes, ByteBuf cb) {
```

Comment:
Initially I wanted a separated method because `writeValues` encode nulls and unset values in a way that is particular to the protocol. But indeed ByteBuffers in payloads are never `null`, and never `== BoundStatement.UNSET` so I guess it's safe to use it. I will change that.